libs/wingc/src/type_check/jsii_importer.rs

```diff
@@ -173,6 +174,14 @@ impl<'a> JsiiImporter<'a> {
 
 		self.setup_namespaces_for(&type_fqn);
 
+		// Hack: if the type is "constructs.IConstruct", we import it manually
+		// this is done so we can avoid loading the constructs module
+		if type_fqn.as_str() == CONSTRUCT_BASE_INTERFACE {
+			let symbol = Symbol::global(type_fqn.type_name());
+			self.register_jsii_type(&type_fqn, &symbol, self.wing_types.anything());
+			return;
+		}
```

Comment:
The hack we are using to avoid loading `constructs.Construct` didn't work out of the box because we have functions in the SDK with parameters accepting `constructs.IConstruct` directly i.e. the type is not just used in `implements`/`extends` clauses.

It _would_ work if I changed all of instances of `IConstruct` in the SDK to `IResource` instead, but that felt like it was worth avoiding.